Recent cleanups

retr0h opened the pull request:
Bring role in line with the changes we have been making to the other
roles.

retr0h committed:
Recent cleanups

Bring role in line with the changes we have been making to the other
roles.

dguerri commented on README.md:
```diff
@@ -22,14 +22,12 @@ Role Variables
-| `rabbit_hostname` | `localhost` | Hostname/IP address where the RabbitMQ service runs ||
-| `rabbit_username` | `rabbit_username_default` | RabbitMQ username for nova ||
-| `rabbit_pass` | `rabbit_pass_default` | RabbitMQ password for nova ||
+| `nova_compute_database_url` | `sqlite:////var/lib/nova/nova.sqlite` | Database URI ||
```
Nova compute doesn't need access to DB. This should be removed.

retr0h committed:
Removed database connection

If one uses conductor, yes we don't need the database url.

retr0h committed:
Start libvirtd

Nova compute requires libvirtd to function.  RHEL doesn't start
services on install list other distros.

retr0h committed:
Added logging

retr0h committed:
Added memcached

dguerri commented on tasks/main.yml:
```diff
@@ -23,6 +23,12 @@
 - include: packages.yml
 - include: configuration.yml
 
+- name: Start libvirtd
```
I think this could be better implemented as an handler triggered by "Install Nova Compute" for RedHat distros.

Apart from that, the change is awesome

retr0h committed:
Reimplemented handler per @dguerri feedback

dguerri commented:
Cool
